mlflow/server/js/src/app.tsx

```diff
@@ -29,9 +32,13 @@ export function MLFlowRoot() {
   return (
     <IntlProvider locale={locale} messages={messages}>
       <Provider store={store}>
-        <DesignSystemContainer>
+        <DesignSystemContainer isDarkTheme={isDarkTheme}>
           <ConfigProvider prefixCls='ant'>
-            {shouldUsePathRouting() ? <MlflowRouter /> : <App />}
+            {shouldUsePathRouting() ? (
+              <MlflowRouter />
```

Comment:
@harupy do you know if I should add the switch to `MlflowHeader.tsx` which is created under `MlflowRouter` here? I'm not sure when `shouldUsePathRouting` is used.

Reply:
Me neither, but adding the switch to the header sounds good to me.

Reply:
oh, just saw this comment, related one: https://github.com/mlflow/mlflow/pull/10169/files#r1381450028